### ISSUE TITLE
TextArea - fix accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#777]
#### Description / User Story

The following incomplete test is thrown by the storybook accessibility addon and should be fixed, if possible:
- Incomplete: Elements must have sufficient color contrast (hasError=true)

![Image](https://github.com/deven-org/B01LER-Kitchen/assets/122102805/c50909a7-871b-4cfe-91ea-a4489b9cde40)

The issue seems to be related to the name prop.

----
#### Requirements
- [x] #540 

----
#### Acceptance Criteria
- [ ] All errors and incomplete tests thrown by the accessibility addon are resolved:
   - [ ] Incomplete: Elements must have sufficient color contrast (readonly=true / hasError=true /hasCounter=true & size=md)

----
#### Background information
- This issue might be similar to #776 and is just an issue of the a11y addon
